templates/bitcoin-sample.conf

```diff
@@ -13,20 +13,21 @@ rest=1
 # Low bandwidth/CPU optimizations
 blocksonly=1
 maxconnections=8
-dbcache=1000
+dbcache=<dbcache>
 maxmempool=300
 maxuploadtarget=5000
 
 # uncomment test if testnet=1 is activated
 #[test]
```

Comment:
Agree with your line of thought, but it looks like `[test]` and `[regtest]` blocks are required for some options like `rpcport`, etc to be applied on testnet and regtest ([link](https://github.com/bitcoin/bitcoin/blob/0f16212c5931b30f430014caa485de53f9a14920/share/examples/bitcoin.conf#L161)).

Reply:
Ahh, ok, then maybe we should not have these in the final conf files unless it's explicitly needed for that version, and then just have it as the first line so everything is covered under it.

Alternatively we don't comment it out and leave all the different network blocks in and hardcode the values, since I assume the network setting will switch which block is used.

This makes the configure script much simpler, since most stuff can be hardcoded in the appropriate block and we toggle between them with `regtest=1`/`testnet=1` etc.

Reply:
This change maybe doesn't need to be part of this PR. We can do it as part fo the planned config cleanup.

Reply:
Yeah, I agree with:

> Alternatively we don't comment it out and leave all the different network blocks in and hardcode the values, since I assume the network setting will switch which block is used.

And:

> We can do it as part of the planned config cleanup.